docker-mesa-src/Dockerfile

```diff
@@ -1,21 +1,23 @@
-FROM python:3.7-stretch
+FROM python:3.11-bookworm
 
 ARG MESASDKVERSION=20.3.2
 ARG MESAVERSION=12778
 
 RUN ln -sfv /bin/bash /bin/sh
 
 RUN apt-get update &&\
-    apt-get install -y binutils bzip2 libc-dev libx11-dev libz-dev make subversion wget &&\
+    apt-get install -y binutils bzip2 libc-dev libx11-dev libz-dev make wget &&\
     apt-get clean -y &&\
     rm -rf /var/lib/apt/lists/* &&\
     truncate -s 0 /var/log/*log
 
-RUN wget --no-verbose -U "" -O /mesasdk.tar.gz http://www.astro.wisc.edu/~townsend/resource/download/mesasdk/mesasdk-x86_64-linux-${MESASDKVERSION}.tar.gz &&\
+RUN wget --no-verbose -U "" -O /mesasdk.tar.gz http://user.astro.wisc.edu/~townsend/resource/download/mesasdk/mesasdk-x86_64-linux-${MESASDKVERSION}.tar.gz &&\
     tar -xvf /mesasdk.tar.gz
-#     rm /mesasdk.tar.gz
+RUN rm /mesasdk.tar.gz
```

Comment:
This do NOT save space, it just makes a new layer which virtually deletes the file

Reply:
Have deleted this command.